src/testdata/ledger-close-meta-v1-protocol-20.json

```diff
@@ -3,13 +3,7 @@
         "v": 1,
         "v1": {
             "ext": {
-                "v": 1,
```

Comment:
Not a big deal at the moment, but wouldn't it make sense to just set the `EMIT_*` flags in the `meta stream contains reasonable meta` test instead?

Reply:
Yeah, I've considered that, but I think rolling back the meta conveys the purpose of the PR better (i.e. we actually _don't_ want to emit any new meta by default). The test actually failed due to uninitialized variable, so I think it's a good thing that we rolled back.